src/main_menu.cpp

```diff
@@ -92,34 +111,155 @@ static int utf8_width_notags( const char *s )
-                                  const std::vector<std::string> &vItems,
-                                  size_t iSel, point offset, int spacing )
+std::vector<int> main_menu::print_menu_items( const catacurses::window &w_in,
+        const std::vector<std::string> &vItems,
+        size_t iSel, point offset, int spacing, bool main )
 {
+    const point win_offset( getbegx( w_in ), getbegy( w_in ) );
+    std::vector<int> ret;
     std::string text;
     for( size_t i = 0; i < vItems.size(); ++i ) {
         if( i > 0 ) {
             text += std::string( spacing, ' ' );
         }
+        ret.push_back( utf8_width_notags( text.c_str() ) );
 
-        std::string temp = shortcut_text( c_white, vItems[i] );
-        if( iSel == i ) {
-            text += string_format( "[%s]", colorize( remove_color_tags( temp ), h_white ) );
-        } else {
-            text += string_format( "[%s]", temp );
-        }
+        std::string temp = shortcut_text( iSel == i ? hilite( c_yellow ) : c_yellow, vItems[i] );
+        text += string_format( "[%s]", colorize( temp, iSel == i ? hilite( c_white ) : c_white ) );
     }
 
     int text_width = utf8_width_notags( text.c_str() );
     if( text_width > getmaxx( w_in ) ) {
         offset.y -= std::ceil( text_width / getmaxx( w_in ) );
     }
 
-    fold_and_print( w_in, offset, getmaxx( w_in ), c_light_gray, text, ']' );
+    std::vector<std::string> menu_txt = foldstring( text, getmaxx( w_in ), ']' );
+
+    int y_off = 0;
+    int sel_opt = 0;
+    for( const std::string &txt : menu_txt ) {
+        trim_and_print( w_in, offset + point( 0, y_off ), getmaxx( w_in ), c_white, txt );
+        if( !main ) {
+            y_off++;
+            continue;
+        }
+        std::vector<std::string> tmp_chars = utf8_display_split( remove_color_tags( txt ) );
+        for( int x = 0; static_cast<size_t>( x ) < tmp_chars.size(); x++ ) {
+            if( tmp_chars.at( x ) == "[" ) {
+                for( int x2 = x; static_cast<size_t>( x2 ) < tmp_chars.size(); x2++ ) {
+                    if( tmp_chars.at( x2 ) == "]" ) {
+                        inclusive_rectangle<point> rec( win_offset + offset + point( x, y_off ),
+                                                        win_offset + offset + point( x2, y_off ) );
+                        main_menu_button_map.emplace_back( rec, sel_opt++ );
+                        break;
+                    }
+                }
+            }
+        }
+        y_off++;
+    }
+
+    return ret;
 }
 
-void main_menu::print_menu( const catacurses::window &w_open, int iSel, const point &offset )
+void main_menu::display_sub_menu( int sel, const point &bottom_left, int sel_line )
 {
+    main_menu_sub_button_map.clear();
+    std::vector<std::string> sub_opts;
+    int xlen = 0;
+    main_menu_opts sel_o = static_cast<main_menu_opts>( sel );
+    switch( sel_o ) {
+        case main_menu_opts::CREDITS:
+            display_text( mmenu_credits, "Credits", sel_line );
```

Comment:
Is "Credits" here translated?

Reply:
Nope, but it's strange that it's not translated in master either:

https://github.com/CleverRaven/Cataclysm-DDA/blob/556afb9bf7cceca4db5a43c0dcac3a10fd2439bc/src/main_menu.cpp#L445-L449